### ISSUE TITLE
L5.1 Call to undefined method Closure::__set_state()

### ISSUE THREAD

**Issue by devtac-agarcia** [thinhbuzz/laravel-google-captcha#22]
Hi guys,

I'm working on laravel 5.1 and after updating the laravel-google-captcha to v2.1.6, I got an error on running `php artisan` command
![image](https://user-images.githubusercontent.com/36835479/36655730-761f0340-1aff-11e8-8fa7-8f86df34bb51.png)
I checked the `bootstrap/cache/config.php` and I got the code below:
![image](https://user-images.githubusercontent.com/36835479/36655825-fde6bc46-1aff-11e8-8899-a8a29124ac45.png)

Not sure what's wrong. Please advise.

Thanks and regards!

**Comment by thinhbuzz** [comment]
please show captcha config

**Comment by devtac-agarcia** [comment]
Hi @thinhbuzz ,

Here's the code in `vendor/buzz/laravel-google-captcha/config/config.php` file, also I can confirm the `\ReCaptcha\RequestMethod\Post` does exist
![image](https://user-images.githubusercontent.com/36835479/36658678-a3698444-1b0c-11e8-839b-e225551c19d2.png)

Regards!

**Comment by thinhbuzz** [comment]
you can try clear cache with command:
`php artisan config:clear`
`php artisan clear-compiled`
`php artisan cache:clear`

**Comment by devtac-agarcia** [comment]
Hi @thinhbuzz ,

It works if I deleted the `bootstrap/cache/config.php` file using `php artisan config:clear` command. But is there any way to retain the `bootstrap/cache/config.php` file? 

I believe the error is the `get_request_method` key which has `Closure::__set_state` value.
```
// bootstrap/cache/config.php
'captcha' => 
  array (
    'secret' => '<secret-here>',
    'sitekey' => '<sitekey-here>',
    'get_request_method' => 
    Closure::__set_state(array(
    )),
  ),
```

Thanks!

**Comment by thinhbuzz** [comment]
ok, i see [ConfigCacheCommand.php#L56](https://github.com/laravel/framework/blob/5.1/src/Illuminate/Foundation/Console/ConfigCacheCommand.php#L56) use `var_export` and this function cannot keep function

**Comment by thinhbuzz** [comment]
issue fixed in commit 09c1f82132967c6d21682eb8806366f43bd414fd

**Comment by devtac-agarcia** [comment]
Thank you @thinhbuzz :+1:
